syntax = "proto3";

//package human_pose;


service TransmitData {
  rpc DoTransmit(DataRequest) returns (DataResponse){}
}
message DataRequest {
  bytes img_nparray = 1;
}
message DataResponse{
  bytes people = 1;
}